cnd/src/swap_protocols/state_store.rs

```diff
@@ -144,9 +131,10 @@ mod tests {
     fn insert_and_get_state() {
         let state_store = InMemoryStateStore::default();
 
-        let bitcoin_pub_key = "02c2a8efce029526d364c2cf39d89e3cdda05e5df7b2cbfc098b4e3d02b70b5275"
-            .parse()
-            .unwrap();
+        let bitcoin_pub_key: identity::Bitcoin =
+            "02c2a8efce029526d364c2cf39d89e3cdda05e5df7b2cbfc098b4e3d02b70b5275"
+                .parse()
+                .unwrap();
```

Comment:
```suggestion
        let bitcoin_pub_key = identity::Bitcoin::from_str("02c2a8efce029526d364c2cf39d89e3cdda05e5df7b2cbfc098b4e3d02b70b5275").unwrap();
```

I feel there is not much point using `.parse()` if you have to hint the type. Personal preference.

Reply:
I agree :smile: